geode-management/src/main/resources/org/apache/geode/management/internal/sanctioned-geode-management-serializables.txt

```diff
@@ -6,7 +6,7 @@ org/apache/geode/management/api/RealizationResult,false,memberName:java/lang/Str
 org/apache/geode/management/configuration/AbstractConfiguration,true,-6612840641128145954
 org/apache/geode/management/configuration/AutoSerializer,true,1,patterns:java/util/List,portable:java/lang/Boolean
 org/apache/geode/management/configuration/ClassName,true,1,className:java/lang/String,initProperties:java/util/Properties
-org/apache/geode/management/configuration/Deployment,true,6992732279452865384,deployedBy:java/lang/String,deployedTime:java/lang/String,jarFileName:java/lang/String
+org/apache/geode/management/configuration/Deployment,true,6992732279452865384,dependencies:java/util/List,deployedBy:java/lang/String,deployedTime:java/lang/String,jarFileName:java/lang/String
```

Comment:
would change this cause any upgrade issue? 'cause I seem to remember this is saved in an internal region, see `Configuration` class which holds reference to this class. See `ConfigurationCompatibilityTest` for example.

Reply:
We should update the serialVersionUID, but with that, it should be able to deserialize old versions of Deployment; dependencies will just be null.